internal/docstore/driver/driver.go

```diff
@@ -104,10 +103,21 @@ type Filter struct {
 	Value     interface{} // the value to compare using the operation
```

Comment:
Is there somewhere where we explain what the operators mean for different types? For example, what does it mean for a struct to be > another struct?

Reply:
Can you document that here?